connectors/v2/youtube.js

```diff
@@ -0,0 +1,87 @@
+'use strict';
+
+/* global Connector */
+
+var scrobbleMusicOnly = false;
+chrome.storage.local.get('Connectors', function(data) {
```

Comment:
I believe not, as the connector is run content-side, so does not have access to the background environment objects like ChromeStorage.